python/salpytools/salpylib.py

```diff
@@ -465,8 +469,10 @@ def __init__(self, Device, sleeptime=1, timeout=5, threadID=1):
         LOGGER.info("Loading Device: {}".format(self.Device))
         # Load SALPY_lib into the class
         self.SALPY_lib = load_SALPYlib(self.Device)
-        # And the code for the SAL__CMD_COMPLETE from the library itself
+        # Shorcuts the code for some the SAL__CMD_XXXXX from the library itself
```

Comment:
Saving 10 characters on use by adding two lines and possible spelling errors here does not seem like a great tradeoff.